frontend/document/koptinterface.lua

```diff
@@ -833,7 +833,9 @@ function KoptInterface:getNativeOCRWord(doc, pageno, rect)
             kc.getTOCRWord, kc, "src",
             0, 0, word_w, word_h,
             self.tessocr_data, self.ocr_lang, self.ocr_type, 0, 1)
-        DocCache:insert(hash, CacheItem:new{ ocrword = word, size = #word + 64 }) -- estimation
+        if word then
```

Comment:
Hi @Frenzie 

Thanks for putting this fix in place! I was having a crash on this exact line and this fixes it 👍🏼 🎉